archetypes/data-playground.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+subtitle: "add subtitle here" 
+draft: true
+image: img/data-playground/
+link: "add github link"
+description: 'Add Description Here'
+license: "Add License Here"
+producer: "Add Producer Here"
```

Comment:
Is the subtitle the type? For example, `subtitle: Stock Market Data`? Should a link to the producer's website be included. I currently see a property for the GitHub link. 

Also, the description on the detail view is different from what's on the list page. Should a `summary` property be added for what will be displayed on the list page?